lib/p5.play.js

```diff
@@ -1537,7 +1536,8 @@ function Sprite(pInst, _x, _y, _w, _h) {
       this.collider = p5.OrientedBoundingBoxCollider.createFromSprite(this, offset, width, height, radians(rotation));
     }
 
-    quadTree.insert(this);
+    // Disabled for Code.org, since perf seems better without the quadtree:
+    // quadTree.insert(this);
```

Comment:
I'm disabling the one remaining call to `quadTree.insert` because it was taking up a moderate chunk of frame time whenever a sprite's collider is created, _even though_ we've disabled the quadtree in Game Lab.

I noticed this by looking closely at the first `draw()` call of the demo app, where all 500 sprites create default colliders the first time they are asked to check for collision with the screen edges.

Presumably this is to maintain internal consistency so you could turn the quadtree on and off at will, though I'm not sure why you'd want to do so.  I'm not going to tear the quadtree out entirely yet (it's still possible other fixes will make it an improvement instead of a detriment down the road) but for now skipping this step saves a few milliseconds per collider creation.